webapp/components/Registration/CreateUserAccount.vue

```diff
@@ -84,9 +87,24 @@ import gql from 'graphql-tag'
 import PasswordStrength from '../Password/Strength'
 import { SweetalertIcon } from 'vue-sweetalert-icons'
 import PasswordForm from '~/components/utils/PasswordFormHelper'
+import { VERSION } from '~/pages/terms-and-conditions'
+
+/* TODO: hier muss die version rein */
 export const SignupVerificationMutation = gql`
-  mutation($nonce: String!, $name: String!, $email: String!, $password: String!) {
-    SignupVerification(nonce: $nonce, email: $email, name: $name, password: $password) {
+  mutation(
+    $nonce: String!
+    $name: String!
+    $email: String!
+    $password: String!
+    $termsAndConditionsAgreedVersion: String!
```

Comment:
Has the exclamation mark to be removed here as well?!
… after @roschaefer suggestion?

Reply:
This is an **input** param! So if the user creates the account, he/she has to tell us which version of the terms and conditions he/she agreed to. This is *different* from the scenario, that we import the database (version is `undefined`) and the server should respond with an object of type `User` and unfortunately that is a user who has not agreed to any terms and conditions. If the server would respond `undefined` for `User.termsAndConditions` the server would crash.

Does that make sense? @Tirokk So the exclamation mark here is correct.

Reply:
Actually, the server will still crash. Ie. if you request `termsAndConditionsAgreedVersion` and we haven't used `neo4jgraphqljs` in the resolver. It is a rare situation though.